src/settings/settings.go

```diff
@@ -58,7 +58,7 @@ type Settings struct {
 	RedisPerSecondPipelineLimit  int           `envconfig:"REDIS_PERSECOND_PIPELINE_LIMIT" default:"0"`
 
 	// Memcache settings
-	MemcacheHostPort string `envconfig:"MEMCACHE_HOST_PORT" default:""`
+	MemcacheUrl string `envconfig:"MEMCACHE_URL" default:""`
```

Comment:
You should be able to change the type here to `[]string` and avoid the need to call `strings.Split` later

https://github.com/kelseyhightower/envconfig has some detail about what the envconfig library supports